### PR TITLE
[CI] List points of contact in ci-basic-overlay.sh

### PR DESCRIPTION
**Pull request by proux01**
Following [yesterday's weekly call](https://github.com/coq/coq/wiki/Coq-Call-2023-11-14), it appears it would be nice to list points of contact for each CI dev.

### CODE REVIEW

**Comment by proux01** [comment]
@affeldt-aist, @aleksnanevski, @andres-erbsen, @arthuraa, @bcpierce00, @benediktahrens, @beta-ziliani, @bgregoir, @Blaisorblade, @clayrat, @CohenCyril, @co-dan, @damien-pous, @ejgallego\
, @erikmd, @fajb, @fpottier, @gares, @gmalecha, @huynhtrankhanh, @Janno, @JasonGross, @jellooo038, @jim-portegies, @Karmaki, @LasseBlaauwbroek, @Lysxia, @liyishuai, @lukaszcz, @mattam82\
, @maximedenes, @minkiminki, @mrhaandi, @nmvdw, @palmskog, @pi8027, @RalfJung, @rmatthes, @samuelgruetter, @silene, @SkySkimmer, @spitters, @strub, @tchajed, @thery, @upamanyus, @vbgl, \
@whonore, @xavierleroy, @ybertot, @yforster  sorry to bother you, we'd like to maintain points of contact for each dev in Coq CI and you seem to be a regular maintainer of one of them, please complain if this is not the case (you can check to which dev your login is related there: https://github.com/coq/coq/pull/18321/files )

**Comment by fpottier** [comment]
Indeed, @jhjourdan and I are the maintainers of `coq-menhirlib`.

File: dev/ci/ci-basic-overlay.sh [hunk 1]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # Deriving
 ########################################################################
 project deriving "https://github.com/arthuraa/deriving" "master"
+# Contact @arthuraa on github
 
 ########################################################################
 # VsCoq
 ########################################################################
 project vscoq "https://github.com/coq-community/vscoq" "coq-master"
+# Contact @maximedenes, @huynhtrankhanh, @thery, @Blaisorblade on github
```
Comment (palmskog) [inline dev/ci/ci-basic-overlay.sh:445]:
```suggestion
# Contact @rtetley on github
```
I think for this part of VsCoq it's only @rtetley these days, maybe @gares as well?

**Comment by erikmd** [comment]
Cc @yforster FYI (because it seems @proux01's comment did not trigger a **mention** for your username)

**Review by SkySkimmer — APPROVED** [review]

File: dev/ci/ci-basic-overlay.sh [hunk 2]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
 # NB: stdpp and Iris refs are gotten from the opam files in the Iris
 # and lambdaRust repos respectively.
 project stdpp "https://gitlab.mpi-sws.org/iris/stdpp" ""
+# Contact @RalfJung on github
 
 project iris "https://gitlab.mpi-sws.org/iris/iris" ""
+# Contact @RalfJung on github
```
Comment (RalfJung) [inline dev/ci/ci-basic-overlay.sh:105]: You can also add @robbertkrebbers  (for Iris and std++ and iris_examples)

File: dev/ci/ci-basic-overlay.sh [hunk 3]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # ATBR
 ########################################################################
 project atbr "https://github.com/coq-community/atbr" "master"
+# Contact @palmskog, @tchajed on github
 
 ########################################################################
 # perennial
 ########################################################################
 project perennial "https://github.com/mit-pdos/perennial" "coq/tested"
+# Contact @upamanyus, @RalfJung on github
```
Comment (RalfJung) [inline dev/ci/ci-basic-overlay.sh:415]: @tchajed could also help with Perennial issues

File: dev/ci/ci-basic-overlay.sh [hunk 4]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # VST
 ########################################################################
 project vst "https://github.com/PrincetonUniversity/VST" "master"
+# Contact @andrew-appel on github
 
 ########################################################################
 # cross-crypto
 ########################################################################
 project cross_crypto "https://github.com/mit-plv/cross-crypto" "master"
+# Contact @andres-erbsen, @JasonGross on github
```
Comment (JasonGross) [inline dev/ci/ci-basic-overlay.sh:165]:
```suggestion
# Contact @andres-erbsen on github
```
I don't have merge bits anymore on this repo

File: dev/ci/ci-basic-overlay.sh [hunk 5]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # coqprime
 ########################################################################
 project coqprime "https://github.com/thery/coqprime" "master"
+# Contact @thery on github
 
 ########################################################################
 # bbv
 ########################################################################
 project bbv "https://github.com/mit-plv/bbv" "master"
+# Contact @JasonGross on github
```
Comment (JasonGross) [inline dev/ci/ci-basic-overlay.sh:225]:
```suggestion
# Contact @JasonGross, @samuelgruetter on github
```

File: dev/ci/ci-basic-overlay.sh [hunk 6]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 
 ########################################################################
 # bedrock2
 ########################################################################
 project bedrock2 "https://github.com/mit-plv/bedrock2" "tested"
+# Contact @samuelgruetter on github
```
Comment (JasonGross) [inline dev/ci/ci-basic-overlay.sh:231]:
```suggestion
# Contact @samuelgruetter, @andres-erbsen on github
```

**Review by JasonGross — DISMISSED** [review]

**Review by nmvdw — APPROVED** [review]

File: dev/ci/ci-basic-overlay.sh [hunk 7]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # ext-lib
 ########################################################################
 project ext_lib "https://github.com/coq-community/coq-ext-lib" "master"
+# Contact @gmalecha, @liyishuai on github
 
 ########################################################################
 # simple-io
 ########################################################################
 project simple_io "https://github.com/Lysxia/coq-simple-io" "master"
+# Contact @Lysxia on github
```
Comment (liyishuai) [inline dev/ci/ci-basic-overlay.sh:282]:
```suggestion
# Contact @Lysxia @liyishuai on github
```

File: dev/ci/ci-basic-overlay.sh [hunk 8]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 
 ########################################################################
 # quickchick
 ########################################################################
 project quickchick "https://github.com/QuickChick/QuickChick" "master"
+# Contact @liyishuai on github
```
Comment (liyishuai) [inline dev/ci/ci-basic-overlay.sh:288]:
```suggestion
# Contact @lemonidas @Lysxia @liyishuai on github
```

File: dev/ci/ci-basic-overlay.sh [hunk 9]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # metacoq
 ########################################################################
 project metacoq "https://github.com/MetaCoq/metacoq" "main"
+# Contact @mattam82, @yforster on github
 
 ########################################################################
 # SF suite
 ########################################################################
 project sf "https://github.com/DeepSpec/sf" "master"
+# Contact @bcpierce00 on github
```
Comment (liyishuai) [inline dev/ci/ci-basic-overlay.sh:427]:
```suggestion
# Contact @bcpierce00 @liyishuai on github
```

**Review by liyishuai — CHANGES_REQUESTED** [review]

**Review by gares — APPROVED** [review]

**Review by LasseBlaauwbroek — APPROVED** [review]

File: dev/ci/ci-basic-overlay.sh [hunk 10]
```diff
@@ -86,345 +99,419 @@ project corn "https://github.com/coq-community/corn" "master"
...
 # Stalmarck
 ########################################################################
 project stalmarck "https://github.com/coq-community/stalmarck" "master"
+# Contact @palmskog on github
 
 ########################################################################
 # coq-library-undecidability
 ########################################################################
 project coq_library_undecidability "https://github.com/uds-psl/coq-library-undecidability" "master"
+# Contact @mrhaandi, yforster on github
```
Comment (mrhaandi) [inline dev/ci/ci-basic-overlay.sh:499]: it should say @yforster

**Review by liyishuai — APPROVED** [review]

File: dev/ci/ci-basic-overlay.sh [hunk 11]
```diff
@@ -43,41 +43,54 @@ function project {
...
+# Contact @CohenCyril on github
 
 project analysis "https://github.com/math-comp/analysis" "master"
+# Contact @affeldt-aist, @CohenCyril on github
 
 ########################################################################
 # UniMath
 ########################################################################
 project unimath "https://github.com/UniMath/UniMath" "master"
+# Contact @benediktahrens, @nmvdw, @rmatthes on github
```
Comment (rmatthes) [inline dev/ci/ci-basic-overlay.sh:75]:
```suggestion
# Contact @benediktahrens, @m-lindgren, @nmvdw, @rmatthes on github
```

**Review by rmatthes — CHANGES_REQUESTED** [review]

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**